parent/pom.xml

```diff
@@ -643,7 +617,7 @@
 
         <carbon.utils.version>1.2.1</carbon.utils.version>
         <carbon.utils.package.import.version.range>[1.0.0, 2.0.0)</carbon.utils.package.import.version.range>
-        <carbon.config.version>2.0.0</carbon.config.version>
+        <carbon.config.version>2.0.1-SNAPSHOT</carbon.config.version>
```

Comment:
yes, we need to release carbon-config before merging this PR. need to change the version after releasing the config